.github/workflows/release.yml

```diff
@@ -66,31 +75,34 @@ jobs:
       - name: Setup PostgreSQL
         uses: ikalnytskyi/action-setup-postgres@v6
 
-      - uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --release
-
       - id: build
         run: |
           rustc --print cfg | grep = > rustc.vars
           source rustc.vars
 
-          pushd target/release
-          if [ "$RUNNER_OS" == "Windows" ]; then
-            export ASSET_NAME="xsnippet-api-${target_arch}-${target_os}.exe.7z"
+          export HOST="${target_arch}-${target_os}-${target_env}"
```

Comment:
Q: I don't get it, who sets this shell variables? Or is this a part of the environment?